lib/internal/Magento/Framework/Stdlib/ArrayManager.php

```diff
@@ -49,7 +49,7 @@ public function exists($path, array $data, $delimiter = self::DEFAULT_PATH_DELIM
      *
      * @param array|string $path
      * @param array $data
-     * @param null $defaultValue
+     * @param array|null $defaultValue
```

Comment:
should not it be `mixed` ?

Reply:
Hello @xmav 
Thank you for your review.
You are right.
It should be only `mixed` because `mixed` contains `null`

@SilinMykola 
Please fix it

Reply:
@xmav thanks for your suggestions! I changed a declaration.